django/db/backends/sqlite3/base.py

```diff
@@ -27,13 +30,30 @@
 from .schema import DatabaseSchemaEditor                    # isort:skip
 
 
+if settings.DEBUG:
```

Comment:
Do we want to hook to `django.core.signals.settings_changed` here?

Reply:
Sounds very sensible!